### DISCUSSION TITLE
Integration stopped working?

### DISCUSSION THREAD

**Discussion by jacontre-c** [guerrerotook/securitas-direct-new-api#106] [General]
hi

Since last night,  alarm was showing unavailable, HA restart did not help

I tried doing integration update, reconfigure, etc, but in all cases, the alarm panel was not showing up
When configuring, the system sends a pin to the mobile phone, it adds the panel (username ) but control panel never shows up

The logs only  show "Unable to find services.yaml for the securitas integration"

Anything that has changed recently?

regards

**Comment by Xalfen** [comment]
Same, I think it's the same issue in #101

**Comment by javcon** [comment]
hi

symptom seems similar... although problem is this was working, and without
changes, it just stopped, reinstall did not help

Also, I am not getting any errors in the HA logs

regards




On Thu, Feb 16, 2023 at 9:23 AM Xalfen ***@***.***> wrote:

> Same, I think it's the same issue in #101
> <https://github.com/guerrerotook/securitas-direct-new-api/discussions/101>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/guerrerotook/securitas-direct-new-api/discussions/106#discussioncomment-4992950>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMDDPVHEMEL554H36YGJRI3WXXPZNANCNFSM6AAAAAAU52F7F4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <guerrerotook/securitas-direct-new-api/repo-discussions/106/comments/4992950
> @github.com>
>


-- 
----
Javier Contreras Albesa

**Comment by Xalfen** [comment]
Managed to get the same error as you after reinstalling then restart HA. I'm now getting only
`2023-02-16 09:53:27.720 WARNING (SyncWorker_3) [homeassistant.helpers.service] Unable to find services.yaml for the securitas integration`
